Tests/MacroToolkitTests/ModifierTests.swift

```diff
@@ -0,0 +1,54 @@
+import XCTest
+import SwiftSyntax
+import SwiftSyntaxBuilder
+@testable import MacroToolkit
+
+final class ModifierTests: XCTestCase {
+    func testAccessModifierInit() {
+        XCTAssertEqual(AccessModifier(rawValue: .keyword(.private)), .private)
```

Comment:
Probably don't need to test every case cause otherwise this test is no different to the source code that defines the behaviour (just simple case matching). But I'll leave that to your discretion, I won't claim to have tons of unit testing experience 👍

Reply:
Agreed. I have removed some. Balancing avoiding redundancy and a reasonable level of test coverage is tough, and something I do poorly.